### ISSUE TITLE
Fix S2479 FN: Support raw string literals with interpolation

### ISSUE THREAD

**Issue by gregory-paidis-sonarsource** [SonarSource/sonar-dotnet#6290]
### Description

FN for S2479 (ControlCharacterInString).

Using C#11 raw string literals, the rule is not raising an issue.

### Repro steps

```csharp
        public const string RawStringLiteralsNonCompliant = """test"""; // Noncompliant
        public const string InterpolatedStringNonCompliant = $"""test{RawStringLiteralsNonCompliant}"""; // FN
```

### Expected behavior

The rule should raise an issue, since the interpolated part is noncompliant as well.

### Actual behavior

The rule does not raise an issue.

**Comment by gregory-paidis-sonarsource** [comment]
This can be marked as resolved after PR #6345 gets merged.

For more details, read [this comment](https://github.com/SonarSource/sonar-dotnet/pull/6345#issuecomment-1310411544)

**Comment by gregory-paidis-sonarsource** [comment]
After discussing this internally with @csaba-sagi-sonarsource , we decided to ignore raw string literals in any form (simple, interpolated, UTF-8) for this rule, since they have no escape mechanism.

This means the following:

```csharp
        public const string simple = """�test"""; // Compliant
        public const string interpolated = $"""test{simple}"""; // Compliant
        public ReadOnlySpan<byte> utf8 = """test"""u8;  // Compliant
```
